modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/StaticHtml2Generator.java

```diff
@@ -195,6 +199,15 @@ public CodegenOperation fromOperation(String path, String httpMethod, Operation
         return op;
     }
 
+    private void preparHtmlForGlobalDescription(Swagger swagger) {
```

Comment:
@JuH33 thanks for the PR. It would be nice to add a 1-liner or a proper docstring explaining what this function does.